File: src/dto/investimentodto.ts
import {UserDTO} from './userdto';
import {SchedinaDTO} from './schedinadto';

export class InvestimentoDTO {
 
 id: number;

 user: UserDTO;

 schedina: SchedinaDTO;

 budget: number;

 durata: number;

 rischio: number;

}

File: src/main/java/it/contrader/dto/InvestimentoDTO.java
package it.contrader.dto;

import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;

import it.contrader.model.Investimento;
import it.contrader.model.Schedina;
import it.contrader.model.Team;
import it.contrader.model.User;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonIdentityInfo(generator=ObjectIdGenerators.PropertyGenerator.class, property="id")

public class InvestimentoDTO {

private long id;
private User user;

private int budget;
private int durata;
private int rischio;


}
